### PR TITLE
Summer slam landing page

### PR DESCRIPTION
**Pull request by illbzo1**
I am reasonably sure this page works as expected, but since I haven't been able to get the site running locally, I'll need someone to take a look and verify everything for me.

I am like a baby.

### CODE REVIEW

File: public/summer-slam-2017/index.html [hunk 1]
```diff
@@ -0,0 +1,539 @@
...
+                      Yet to be determined, but it'll be delicious. There will be vegetarian options.
+                    </p>
+                  </li>
+                  <!-- /Item -->
+
+                  <!-- Item -->
+                  <li>
+                    <h3>Drink</h3>
+                    <p class="icon-1 icon-1-6">
+                      Yet to be determined, but it'll be local. It will also probably be vegetarian. I assume.
```
Comment (netshade) [inline public/summer-slam-2017/index.html]: Add specific mention of non-alcoholic drinks here I think, a la:

Drink: Lemonade, Soda, Beer or Wine. We'll keep you hydrated and happy.

File: public/summer-slam-2017/index.html [hunk 2]
```diff
@@ -0,0 +1,539 @@
...
+                    </p>
+                  </li>
+                  <!-- /Item -->
+
+                  <!-- Item -->
+                  <li>
+                    <h3>Music</h3>
+                    <p class="icon-1 icon-1-6">
+                      Also yet to be determined. May be local. Could also be vegetarian?
```
Comment (netshade) [inline public/summer-slam-2017/index.html]: Nice. :)

File: public/summer-slam-2017/index.html [hunk 3]
```diff
@@ -0,0 +1,539 @@
...
+
+                  </li>
+                  <!-- /Item -->
+
+                </ul>
+                <!-- /Features list -->
+
+
+                <p class="subtitle-paragraph">
+                  For more information on how to become a sponsor, check out <a target="__BLANK" href="/summer-slam-2017/image/indyhackers.org_holiday_social_2016_sponsorships.pdf">the Sponsorship PDF</a> and/or just <a target="__BLANK" href="mailto:sponsors@indyhackers.org?subject=Sponsoring-Indy-Hackers-Holiday-Social">e-mail us</a>! Absolutely all sponsor money goes toward the event.
```
Comment (netshade) [inline public/summer-slam-2017/index.html]: This refers to the winter 2016 sponsor sheet, should likely update that as well.

**Comment by illbzo1** [comment]
Yeah, good call. Want to keep this updated once more details are solidified.       
  
  
  
  
  

  
  
>

File: public/summer-slam-2017/index.html [hunk 4]
```diff
@@ -0,0 +1,539 @@
...
+
+                <h2> I am a hacker...</h2>
+                <!-- Features list -->
+                <ul class="no-list list-2 clear-fix">
+
+                  <!-- Item -->
+                  <li>
+                    <h3>Want to do more with IndyHackers?</h3>
+                    <p class="icon-1 icon-1-6">
+                      This is all acomplished because a few folks really love the local community, and wanted a way to bring it together. Do you have ideas about how we can make this better? We would love to talk to you! Come chat with us on the <a href="https://www.flowdock.com/invitations/098c86c1494263de8002f9b6b2698d0fde1cb352" target="__BLANK">Indy Hackers Flowdock flow</a>
```
Comment (netshade) [inline public/summer-slam-2017/index.html]: Should link to the Slack I expect.

File: public/summer-slam-2017/index.html (hunk 3 above)
Comment (mileszs) [inline public/summer-slam-2017/index.html]: I haven't made this thing yet. Should probably start a discussion about it. I think we could remove the sponsor sheet thing to just "email us" for now.

**Comment by illbzo1** [comment]
+1 for removing the PDF link for now.       
  
  
  
  
  

  
  
>

File: public/summer-slam-2017/index.html (hunk 1 above)
Comment (illbzo1) [inline public/summer-slam-2017/index.html]: Done, thanks for the update!

File: public/summer-slam-2017/index.html (hunk 3 above)
Comment (illbzo1) [inline public/summer-slam-2017/index.html]: Yep, I've removed the PDF link for now.

File: public/summer-slam-2017/index.html (hunk 4 above)
Comment (illbzo1) [inline public/summer-slam-2017/index.html]: Done!

**Comment by netshade** [comment]
:+1:

**Pull request merged**